### PR TITLE
Append errors if message panel is open

### LINKED ISSUE #1339: Only the last thrown error is visible in our error panel
If the message_view (error panel) is open, each `logger.error` call overwrites the previous message. It would be probably better to append here if the panel is open. (Esp. since errors are only displayed once per session, quickly overwritten errors become otherwise inaccessible)

### PR DESCRIPTION
**Pull request by kaste**
Fixes #1339

### CODE REVIEW

File: message_view.py [hunk 1]
```diff
@@ -11,14 +12,27 @@ def plugin_unloaded():
...
+
+        scroll_to = panel_view.size()
+        msg = msg.rstrip() + '\n\n\n'
+
         panel_view.set_read_only(False)
         panel_view.run_command('append', {'characters': msg})
         panel_view.set_read_only(True)
-        panel_view.show(0)
-        self.window.run_command("show_panel", {"panel": "output.{}".format(PANEL_NAME)})
+        panel_view.show(scroll_to)
```
Comment (braver) [inline message_view.py]: Would that make it scroll as new messages come in? I'm not sure I'd want that.

Comment (kaste) [reply message_view.py]: This looks and feels pretty good. I added this while testing the feature.

Without it you hardly notice the 'append'. And if you really have the error panel open, you kinda wait for a reaction.

Say you try the --foo arg on flake8. Then the panel opens and says nope, there's no foo allowed. You edit to fooz, then save, then the error panel will append the new message and show it.

If you closed before the error will be replaced, and thus will also show up directly, without scrolling.

Comment (braver) [reply message_view.py]: Ok, not a big deal anyway.

Comment (kaste) [reply message_view.py]: The pragmatic braver™️

**Pull request merged**